src/Microsoft.AspNetCore.SignalR.Client.Core/HubConnection.cs

```diff
@@ -49,12 +57,28 @@ public partial class HubConnection
         public TimeSpan ServerTimeout { get; set; } = DefaultServerTimeout;
         public TimeSpan HandshakeTimeout { get; set; } = DefaultHandshakeTimeout;
 
+        /// <summary>
+        /// Initializes a new instance of the <see cref="HubConnection"/> class.
+        /// </summary>
+        /// <param name="connectionFactory">The <see cref="IConnectionFactory" /> used to create a connection each time <see cref="StartAsync" /> is called.</param>
+        /// <param name="protocol">The <see cref="IHubProtocol" /> used by the connection.</param>
+        /// <param name="serviceProvider">An <see cref="IServiceProvider"/> containing the services provided to this connection.</param>
+        /// <param name="loggerFactory">The logger factory.</param>
+        /// <remarks>
+        /// The <see cref="IServiceProvider"/> used to initialize the connection will be disposed when the connection is disposed.
```

Comment:
"connection" is used to refer to `HubConnection` and the underlying `IConnection` in different spots in this xml section. We should be more clear